pkg/client/clientset/controller-versioned/doc.go

```diff
@@ -0,0 +1,7 @@
+// Package controller_versioned provides a set of Appwrapper clients to be used with interacting with AppWrapper CRDS.
+// RC4
+// Deprecated: controller_versioned has been replaced by the versioned as that package is generated using the client gen tool.
+// controller_versioned interfaces and methods should not be used except for compatibility. It will be removed in future versions.
+//
+// This package is frozen and no new functionality will be added.
+package controller_versioned
```

Comment:
I think it's OK to remove the existing client altogether.

Reply:
@asm582, @dmatch01 and @metalcycling do you agree? I have no problems with doing it.

Reply:
Thanks for the PR @z103cb  and review @astefanutti . I am ok as long as the test pass, MCAD deals with unstructured objects to deploy GenericItems, and I hope with client upgrades there are no surprises wrt to marshaling and unmarshalling of unstructured objects.

Reply:
@z103cb If the e2e test fails, we can identify the cause of the failure. if they continue to pass, I vote for merging this PR. testing will be improved over time. please feel free to open up new issues for testing.

Reply:
Ok, will do.